docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
 
-  python:
+  app:
```

Comment:
This change will break the CI

Reply:
well, then i'll propose this change in separated PR